src/main/java/uk/gov/hmcts/et/common/model/ccd/items/GenericTypeItem.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.et.common.model.ccd.items;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@SuperBuilder
+@NoArgsConstructor
+public class GenericTypeItem<T> {
+
+    @JsonProperty("id")
+    private String id;
+    @JsonProperty("value")
+    private T value;
+
+}
```

Comment:
Excellent, hopefully we can use this pattern going forward. Will save a lot of manual work declaring XTypeItems